smart-orchestrator/src/com/axonivy/utils/smart/orchestrator/AgenticProcessCall.java

```diff
@@ -111,18 +112,20 @@ private <T> Optional<T> execute(IIvyScriptContext context, String configKey, Cla
     }
   }
 
+  private Optional<String> expand(IIvyScriptContext context, String confKey) {
+    return new MacroExpander(context).expand(getConfig().get(confKey));
+  }
+
   public static class Editor extends UiEditorExtension {
 
     @Override
     public void initUiFields(ExtensionUiBuilder ui) {
       ui.label("How can I assist you today?").create();
-      ui.scriptField(Conf.QUERY)
+      ui.textField(Conf.QUERY)
```

Comment:
this will also "slightly" affect you @chnam-axonivy  ... the input must no longer be quoted like a string. And references variables within the prompts now require the 'macro' post and prefix.